<launch>

    <!--    copy from arc_star.launch, rosbag or camera? -->
    <!--    如果是camera，是不是应该让他先把celex_ros给启动了-->
    <arg name="rosbag_flag" default="0"/>
    <arg name="rosbag_path" default=""/>

    <!--    get event data from celex5 real time -->
    <group if="$(eval arg('rosbag_flag')==0)">

        <node pkg="celex5_detector" type="celex5_detector" name="celex5_detector_object" output="screen">
            <!--        recv_ed topic-->
            <!--            下边的转进行events和images的修改，注意成对的注释-->
                        <remap from="detector_recved_topic" to="/celex5_mipi/display/binary_img/raw_image"/>
                        <param name="recved_data_type" value="images"/>

<!--            <remap from="detector_recved_topic" to="/celex5_mipi/events"/>-->
<!--            <param name="recved_data_type" value="events"/>-->

<!--            <param name="detector_name" value="cluster"/>-->
            <param name="detector_name" value="morph"/>
        </node>
    </group>


    <!--    get event data from rosbag play bag.bag-->
    <group if="$(eval arg('rosbag_flag')!=0)">
        <node pkg="rosbag" type="play" name="bagplayer" args="$(arg rosbag_path)">

            <param name="test_param" value="testprivate"/>
        </node>
    </group>

    <!--    事件或者是frame的是不是也要放在这里-->


</launch>